### PR TITLE
Update Sentiment Analysis Tutorial to .NET 6

### PR DESCRIPTION
**Pull request by jwood803**
Updates the sentiment analysis tutorial with the ML.NET API to .NET 6.

Updates #26778.

[Preview](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/sentiment-analysis?branch=pr-en-us-26824)

### CODE REVIEW

**Review by luisquintanilla — COMMENTED** [review]
Looks good to me @jwood803. One small change. In the intro para it still says VS 2017

image

File: docs/machine-learning/tutorials/snippets/sentiment-analysis/csharp/SentimentAnalysis.csproj [hunk 1]
```diff
@@ -3,10 +3,11 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
 
   <PropertyGroup>
-    <LangVersion>7.1</LangVersion>
+    <LangVersion>10.0</LangVersion>
   </PropertyGroup>
```
Comment (IEvangelist) [inline docs/machine-learning/tutorials/snippets/sentiment-analysis/csharp/SentimentAnalysis.csproj:11]: This is the default C# version for the `net6.0` TFM, any reason we're calling it out explicitly?

**Review by IEvangelist — COMMENTED** [review]
Hi @luisquintanilla, are you good with these updates?

File: docs/machine-learning/tutorials/snippets/sentiment-analysis/csharp/SentimentAnalysis.csproj (hunk 1 above)
Comment (luisquintanilla) [reply docs/machine-learning/tutorials/snippets/sentiment-analysis/csharp/SentimentAnalysis.csproj:11]: Good question. @jwood803 can you confirm that with the GA version of .NET 6 you don't have to specify it?

Comment (jwood803) [reply docs/machine-learning/tutorials/snippets/sentiment-analysis/csharp/SentimentAnalysis.csproj]: Removed and the build passes 😄

**Review by luisquintanilla — APPROVED** [review]
Looks good to me. Thanks

**Pull request merged**